action/protocol/rewarding/protocol.go

```diff
@@ -215,16 +215,11 @@ func (p *Protocol) Handle(
 		return p.settleUserAction(ctx, sm, uint64(iotextypes.ReceiptStatus_Success), si, nil, rlog)
 	case *action.ClaimFromRewardingFund:
 		si := sm.Snapshot()
-		claimTo := act.Address()
-		if claimTo == "" {
-			claimTo = protocol.MustGetActionCtx(ctx).Caller.String()
+		addr := protocol.MustGetActionCtx(ctx).Caller
+		if protocol.MustGetFeatureCtx(ctx).AddClaimRewardAddress && act.Address() != nil {
```

Comment:
shall we reject an action if `!protocol.MustGetFeatureCtx(ctx).AddClaimRewardAddress && act.Address() != nil`?

Reply:
can check it when validating

Reply:
added